github-for-jira.sd.yml

```diff
@@ -157,7 +157,7 @@ environmentOverrides:
     config:
       environmentVariables:
         APP_URL: https://github-for-jira.dev.services.atlassian.com
-        WEBHOOK_PROXY_URL: https://github-for-jira.dev.services.atlassian.com/github/events
+        WEBHOOK_PROXY_URL: https://github-for-jira.dev.services.atlassian.com
```

Comment:
What's this change? Is the `/github/events` now appended in the code itself?

Reply:
always was, this was just wrong but the code just got the 'domain' part of that url so it worked.